src/apps/content-editor/src/app/components/Editor/Field/Field.js

```diff
@@ -129,17 +129,26 @@ function resolveRelatedOptions(
       );
     })
     .map((itemZUID) => {
-      return {
-        filterValue: items[itemZUID].data[field.name],
-        value: itemZUID,
-        component: (
-          <RelatedOption
-            modelZUID={modelZUID}
-            itemZUID={itemZUID}
-            text={items[itemZUID].data[field.name]}
-          />
-        ),
-      };
+      for (const [key, value] of Object.entries(items[itemZUID].siblings)) {
```

Comment:
This is some fairly involved programming to get the language code. Please add a comment explaining the why and how.